src/components/Menu/MenuItem/__stories__/MenuItem.mdx

```diff
@@ -44,6 +44,12 @@ Use menu item for drawing one options that displayed inside a menu.
   <Story of={MenuItemStories.Label} />
 </Canvas>
 
+### SubMenu
```

Comment:
Nice!
Do we also want to add example of `split` case to the Menu `Menu with sub menu` story? Have you discussed it with Meirav? I think it'll be nice

Reply:
ohhh cool, haven't thought about it.
i'll ask her